cerberus/aws_auth.py

```diff
@@ -36,11 +36,21 @@ def set_auth(self, role_arn=None, region=None):
         sts_client = boto3.client('sts')
 
         if role_arn is None:
-            caller_identity = sts_client.get_caller_identity()
-            account_id = caller_identity.get('Account')
             role_name = self.get_role_name()
-            self.role_arn = caller_identity.get('Arn') if role_name is False \
-                    else "arn:aws:iam::" + account_id + ":role/" + role_name
+
```

Comment:
I worked with Amer on this some yesterday.  It looks right to me.  This is the same pattern we are using in the Java client.

Reply:
@tlisonbee I am planning to do some additional tests this morning.
My Docker image is currently installing client using pip3, so I will have to change it to use git to pull from this repo, and build client locally instead of pip3. I will update you on results.

Reply:
This seems right to me.

Will your testing include a role with a path, as well as a role without a path?

Reply:
Completed testing with role including path and tests were successful.
Continuing testing with role not including path.

Reply:
Both tests when creating client with IAM role with path and role without path are passing.
I have messaged you (@sdford and @tlisonbee) with test results.